endpoints/namada-se/apis.json

```diff
@@ -18,6 +18,10 @@
   {
     "id": "thoale3009",
     "endpoint": "https://namada-indexer.thoalt.com"
+  },
+  {
+    "id": "gruberx_ursus",
+    "endpoint": "https://namada-rpc.ursus.live"
```

Comment:
This should be an indexer endpoint, this is an RPC. Either change or remove it.

Reply:
Sorry my mistake, I'll fix it.